snuba/utils/streams/backends/kafka.py

```diff
@@ -629,6 +629,16 @@ def closed(self) -> bool:
     "sasl.username",
     "sasl.password",
     "security.protocol",
+    "ssl.client.auth",
```

Comment:
This does not seem a librdkafka setting. (https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md) 
What do you plan to use it for ?